.circleci/config.yml

```diff
@@ -0,0 +1,55 @@
+version: 2
+jobs:
+  build_and_test:
```

Comment:
Last nitpick, could we call this step `build`? Due to legacy naming, the default job on CircleCI is called `build`, and if you use their CLI for local testing, that's the only job that is able to run.